awx/api/serializers.py

```
@@ -3323,7 +3285,11 @@ def validate(self, attrs):
 
 class JobTemplateSerializer(JobTemplateMixin, UnifiedJobTemplateSerializer, JobOptionsSerializer):
     show_capabilities = ['start', 'schedule', 'copy', 'edit', 'delete']
-    capabilities_prefetch = ['admin', 'execute', {'copy': ['project.use', 'inventory.use']}]
+    capabilities_prefetch = [
+        {'edit': 'change'},
+        {'start': 'execute_jobtemplate'},
+        {'copy': ['project.use_project', 'inventory.use_inventory']},
+    ]
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)